README.md

```diff
@@ -149,11 +142,5 @@ type InstalledAppInfo = {
 
 ### `on( event, callback )`
 **Events**
-- `sign-in` - emitted when the user completes a successful sign-in
-- `sign-up` - emitted when the user completes a successful sign-up
-- `sign-out` - emitted when the user competes a successful sign-out
-- `canceled` - emitted when the user purposefully exits sign-in/up
 - `signal` - emitted when a signal is passed from chaperone
-- `available` - emitted when the connection to chaperone and envoy are opened and hosted app is ready for zome calls
-- `unavailable` - emitted when the ws connection to chaperone and/or envoy is closed
-- `unrecoverable-agent-state` - emitted when an unrecoverable error event is passed from chapeone
+- `agent-state` - emitted switching to a new agent, availability changes, or on unrecoverable agent state. The event passes one argument, and object of type `AgentState` (see above)
```

Comment:
```suggestion
- `agent-state` - emitted switching to a new agent, availability changes, or on unrecoverable agent state. The event passes one argument, the current value of `this.agent`
```
I think it's worth clarifying that the argument and the field have identical values.